draft-tus-resumable-uploads-protocol.md

```diff
@@ -223,6 +223,7 @@ upload-incomplete: ?1
 
 :status: 201
 upload-incomplete: ?1
+upload-offset: [new offset]
```

Comment:
Let's just add a random `content-length` to the request and matching value here

Reply:
Good idea, done!